build/NuSpecs/Microsoft.WindowsAppSDK.Foundation.props

```
@@ -8,4 +8,14 @@
     <WindowsAppSdkFoundation>true</WindowsAppSdkFoundation>
   </PropertyGroup>
 
+  <!--
+      At least for now, we want MRT Core to not be used in WAPPROJ projects. There, we want the in-box PRI generation tooling to be used.
+      Likewise, we don't want this out-of-box PRI generation tooling to be used with UWPs.
+  -->
+  <PropertyGroup>
+    <EnableCoreMrtTooling Condition="'$(EnableCoreMrtTooling)'=='' and '$(WindowsAppContainer)'!='true' and '$(MSBuildProjectExtension)' != '.wapproj'">true</EnableCoreMrtTooling>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.Windows.ApplicationModel.Resources.props" Condition="Exists('$(MSBuildThisFileDirectory)Microsoft.Windows.ApplicationModel.Resources.props') and '$(EnableCoreMrtTooling)'=='true'" />
```

Nit: in the case of missing files that should always be in your NuGet package/installed/present, it's usually preferable to avoid the `Exists` condition and just let MSBuild error on the import. Otherwise you have to spend time debugging "the project loaded fine but didn't behave right" and eventually figure out that it was because a file was missing.

```suggestion
  <Import Project="$(MSBuildThisFileDirectory)Microsoft.Windows.ApplicationModel.Resources.props" Condition="'$(EnableCoreMrtTooling)'=='true'" />
```

---

Done!